apps/web-mzima-client/src/app/post/post-edit/post-edit.component.html

```diff
@@ -156,12 +162,18 @@ <h2 class="task-label">{{ task.translations[activeLanguage]?.label || task.label
                 [attr.aria-label]="field.label"
                 [formControlName]="field.key"
                 *ngIf="field.options?.length; else noOptions"
-                [data-qa]="'post-item-post-radio' + field.key"
+                [data-qa]="
+                  field.label.replaceAll(' ', '-').replace('(', '').replace(')', '') | lowercase
```

Comment:
I suggest you to create custom helper for this magic mutations since we are using same in a different places @Ifycode

Reply:
Reminder to clean up as requested